docs/oasis-node/rpc.md

```diff
@@ -0,0 +1,97 @@
+# RPC
+
+Oasis Node exposes an RPC interface to enable external applications to query
+current [consensus] and [runtime] states, [submit transactions], etc.
+
+The RPC interface is ONLY exposed via an AF_LOCAL socket called `internal.sock`
+located in the node's data directory. **This interface should NEVER be directly
+exposed over the network as it has no authentication and allows full control,
+including shutdown, of a node.**
+
+[consensus]: ../consensus/index.md
+[runtime]: ../runtime/index.md
+[submit transactions]: ../consensus/transactions.md#submission
+
+## Protocol
+
+As other parts of Oasis Core, the RPC interface exposed by Oasis Node uses the
+[gRPC protocol] with the CBOR codec (instead of Protocol Buffers). If your
+application is written in Go, you can use the convenience gRPC wrappers provided
```

Comment:
Could also link to our rust client crate as an example of a rust gRPC wrapper? (although with a note that it only implements some of the methods). Maybe [client/src/transaction/api/client.rs](https://github.com/oasislabs/oasis-core/blob/master/client/src/transaction/api/client.rs) is a good example

Reply:
Good idea, added.